onnx/onnx.in.proto

```diff
@@ -442,6 +447,21 @@ message TensorProto {
   repeated uint64 uint64_data = 11 [packed = true];
 }
 
+// A serialized sparse-tensor value
+message SparseTensorProto {
+  // The sequence of non-default values are encoded as a tensor of shape [NNZ].
+  // The default-value is zero for numeric tensors, and empty-string for string tensors.
+  optional TensorProto values = 1;
```

Comment:
Should we generalize this and allow values to be a tensor of rank more than 1? That would be useful if we want to capture sparse-tensors with s sparse-dimensions followed by d dense-dimensions where d > 1.

Reply:
I feel yes for reasons mentioned above.